Library/Homebrew/language/node.rb

```diff
@@ -32,13 +44,13 @@ def self.setup_npm_environment
       end
     end
 
-    def self.std_npm_install_args(libexec)
+    def self.std_npm_install_args(libexec, prepublish_requires_deps = false, pack_skip_prepublish = false)
```

Comment:
I think longer arguments makes these much easier to read but agreed on keywords. `req_` generally means `requirement_` in Homebrew. This is why saving a few characters on variable names is generally a false economy as you need to add a comment to explain your translation.